x/tokenfactory/client/cli/query.go

```diff
@@ -0,0 +1,94 @@
+package cli
+
+import (
+	"fmt"
+
+	// "strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+
+	// "github.com/cosmos/cosmos-sdk/client/flags"
+	// sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/osmosis-labs/osmosis/v7/x/tokenfactory/types"
+)
+
+// GetQueryCmd returns the cli query commands for this module
+func GetQueryCmd(queryRoute string) *cobra.Command {
+	// Group tokenfactory queries under a subcommand
+	cmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      fmt.Sprintf("Querying commands for the %s module", types.ModuleName),
```

Comment:
```suggestion
		Short:                      fmt.Sprintf("Querying commands for the %s module", queryRoute),
```

Reply:
I think we should keep types.ModuleName, and delete queryRoute